New version: CryoGrid v0.23.3

JuliaRegistrator opened the pull request:
- Registering package: CryoGrid
- Repository: https://github.com/CryoGrid/CryoGrid.jl
- Created by: @bgroenks96
- Version: v0.23.3
- Commit: aecf327f344c8734a6fffa6937fffbb601dd8dd4
- Reviewed by: @bgroenks96
- Reference: https://github.com/CryoGrid/CryoGrid.jl/commit/aecf327f344c8734a6fffa6937fffbb601dd8dd4#commitcomment-161484227
- Description: Next generation permafrost process modeling in the Julia programming language.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6d4212e23264b0af991fcca2c6863c75503ffc23530f6dfd5d2ce18388bc57b0001ef306c14f641d9e68629665a2b39fe49d7bc8cf8580e3bad7dc4c09ce910ac5c50f03e6b024d9be95e0b185f50c2f2cd3638394f9fdeb4c818fc25b5c4e2a0af47ae533171e19ead3bc59cdd2a8d0ad976a6b9d2fe4d0ba0cd2070dafbe6bb09b8fd2d6b93ed5005b36423886010a2e0a01de6dee7b44c8958ff1cc6c38bb6108e077c80cc0d437ef33ce67096698e -->

bgroenks96 commented:
Can we restart the AutoMerge checks? The error seems to be internal/networking related.

JuliaRegistrator committed:
New version: CryoGrid v0.23.3

UUID: a535b82e-5f3d-4d97-8b0b-d6483f5bebd5
Repo: https://github.com/CryoGrid/CryoGrid.jl.git
Tree: 24ed9bb3817b766df2fe1b24b2c42eeabcfcb4c9

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
